WordPress/twentytwentytwo
# Hide 404 pattern from inserter

**Description**

This PR hides the 404 pattern from the block inserter. I'm not actually sure if we want to do this, but thought I'd open a PR to discuss.

**Testing Instructions** 

- Open the pattern inserter and go to the General patterns
- Verify the 404 pattern does not show up
